locations/k9.yml

```diff
@@ -113,19 +74,37 @@ networks:
   # MGMT
   - vid: 42
     role: mgmt
+    name: omgmt
     prefix: 10.31.9.208/28
     gateway: 1
     dns: 1
     ipv6_subprefix: 1
     assignments:
       k9-core: 1      # 10.31.9.209
       # k9-switch: 2  # 10.31.9.210
```

Comment:
@spolack can the old MGMT be removed?

Reply:
@Noki

Reply:
Yes, it can be removed. Everything is configured to use the new one.